testing-common/src/main/groovy/io/opentelemetry/auto/test/utils/TraceUtils.groovy

```diff
@@ -18,13 +18,14 @@ package io.opentelemetry.auto.test.utils
 
 import io.grpc.Context
 import io.opentelemetry.OpenTelemetry
-import io.opentelemetry.auto.bootstrap.instrumentation.decorator.BaseDecorator
-import io.opentelemetry.auto.bootstrap.instrumentation.decorator.BaseTracer
 import io.opentelemetry.auto.test.asserts.TraceAssert
 import io.opentelemetry.context.Scope
+import io.opentelemetry.instrumentation.library.api.decorator.BaseDecorator
+import io.opentelemetry.instrumentation.library.api.decorator.BaseTracer
```

Comment:
I think `io.opentelemetry.instrumentation.api` would be better package, without `library`

Reply:
👍